ckan-dev/setup/start_ckan_development.sh

```diff
@@ -37,6 +49,20 @@ python prerun.py
 echo "Loading the following plugins: $CKAN__PLUGINS"
 paster --plugin=ckan config-tool $CKAN_INI "ckan.plugins = $CKAN__PLUGINS"
 
+# Update test-core.ini DB, SOLR & Redis settings
+echo "Loading test settings into test-core.ini"
+paster --plugin=ckan config-tool \
+    $SRC_DIR/ckan/test-core.ini "sqlalchemy.url = $TEST_CKAN_SQLALCHEMY_URL"
+paster --plugin=ckan config-tool \
+    $SRC_DIR/ckan/test-core.ini "ckan.datstore.write_url = $TEST_CKAN_DATASTORE_WRITE_URL"
+paster --plugin=ckan config-tool \
+    $SRC_DIR/ckan/test-core.ini "ckan.datstore.read_url = $TEST_CKAN_DATASTORE_READ_URL"
+
+paster --plugin=ckan config-tool $SRC_DIR/ckan/test-core.ini "solr_url = http://solr:8983/solr/ckan"
+paster --plugin=ckan config-tool $SRC_DIR/ckan/test-core.ini "ckan.redis_url = redis://redis:6379/1"
+
+# Run the prerun script to init CKAN and create the default admin user
+python prerun.py
```

Comment:
This is already called on line 46, no need to call it again right?

Reply:
Sorry, got pulled away and forgot to look it over before pushing it out